lib/peep.rb

```diff
@@ -0,0 +1,47 @@
+require "pg"
+
+class Peep
+  attr_reader :id, :message, :date_posted, :keyword
+
+  def initialize(id:, message:, date_posted:)
+    @id = id
+    @message = message
+    @date_posted = date_posted
+  end
+
+  def self.all
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+
+    result = connection.exec("SELECT * FROM peeps ORDER BY date_posted DESC;")
+    result.map do |peep|
+      Peep.new(id: peep["id"], message: peep["message"], date_posted: peep["date_posted"])
+    end
+  end
+
+  def self.create(message:)
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+
+    connection.exec("INSERT INTO peeps (message) VALUES('#{message}')")
```

Comment:
You should look up the SQL Injection walkthrough from the bookmark manager challenge (https://github.com/makersacademy/course/blob/main/apprenticeships_bookmark_manager/walkthroughs/12.md) to make sure that someone can't break your program

With this style of query you've written, someone could write a peep that says 'DROP TABLE' and delete your database!